cognite/powerops/data_classes/shop_file_config.py

```diff
@@ -13,6 +13,7 @@
 class ShopFileConfig(BaseModel):
     watercourse_name: str
     path: Optional[str] = None
```

Comment:
If you make `path` into a `pathlib.Path`, then it is easy to use the filename as part of the `external_id`.